tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -135,28 +132,11 @@
     grid_file = 'INPUT/grid_spec.nc'
 /
 
-
 &gfs_physics_nml
-    sfclay_compute_flux = @[SFCLAY_COMPUTE_FLUX]
```

Comment:
This option must be set to use the mynn surface scheme with the flake model.

Reply:
I second this. This option must not be removed.